source/sgp_mode/Tasks.h

```diff
@@ -117,6 +124,28 @@ class TaskSet {
       n_succeeds_sym.push_back(emp::NewPtr<std::atomic<size_t>>(0));
     }
   }
+  /**
+   * Input: A pointer to an organism's square frequency map, as well as the value of the current output
+   *
+   * Output: None
+   *
+   * Purpose: Adds the output to the SquareMap with a count of 1 if not already included; 
+   * otherwise, increments the count for that output
+   */
+  void IncrementSquareMap(Task &task, CPUState &state, uint32_t output, std::map<uint32_t, uint32_t> &calculationMap){
+            task.curHostOutput = output;
+            if (calculationMap.empty()){//Base case for when the map is empty
+              calculationMap.insert(std::pair<uint32_t, uint32_t>(output, 1));
+            }else{
```

Comment:
Codecov seems to think that you aren't testing the case where there is already something in the map for this function, so you should add such a test